python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -94,7 +123,37 @@ def get_connection(self, raw_conn=True):
                 conn.commit()
             conn.close()
 
-    def execute_query(self, sql, parameters=None, fetch_results=False, use_pandas_result=False):
+    @public
+    def execute_query(
+        self,
+        sql: str,
+        parameters: Optional[Union[Sequence[Any], Mapping[Any, Any]]] = None,
```

Comment:
this type annotation is pulled from the snowflake type annotation https://github.com/snowflakedb/snowflake-connector-python/blob/0b24148ec0c947764c50f093f6bc1fe1638fb551/src/snowflake/connector/cursor.py#L572